topology/probes/k8s/graph.go

```diff
@@ -32,6 +32,8 @@ const (
 	KubeKey = "K8s"
 	// ExtraKey is the metadata area for k8s extra fields
 	ExtraKey = "K8s.Extra"
+	// Cluster name field
```

Comment:
remove comment ... rename to ClusterNameField

Reply:
Done